pkgs/development/python-modules/tqdm/default.nix

```
@@ -7,22 +7,32 @@
 , flake8
 , setuptools_scm
 , pytestCheckHook
+, pytest-asyncio
+, pytest-timeout
 }:
 
 buildPythonPackage rec {
   pname = "tqdm";
-  version = "4.54.1";
+  version = "4.58.0";
 
   src = fetchPypi {
     inherit pname version;
-    sha256 = "1x9chlh3msikddmq8p8p5s5kgqqs48bclxgzz3vb9ygcwjimidiq";
+    sha256 = "1fjvaag1wy70gglxjkfnn0acrya7fbhzi4adbs1bpap8x03wffn2";
   };
 
   nativeBuildInputs = [
     setuptools_scm
   ];
 
-  checkInputs = [ nose coverage glibcLocales flake8 pytestCheckHook ];
+  checkInputs = [
+    nose
+    coverage
+    glibcLocales
+    flake8
```

```suggestion
    coverage
```

Are the removed checkInputs no longer required?

---

No `checkInputs` where removed, just flowed them into multiple lines as the line was getting pretty long

---

I meant to ask if all of them are still required because upstream did not list them as requirement.

---

Sorry, misunderstood you there. You are right, those inputs can be removed. I've also discovered some optional tests which I've also enabled (don't seem particularly expensive).